onadata/libs/utils/logger_tools.py

```diff
@@ -403,14 +404,17 @@ def safe_create_instance(username, xml_file, media_files, uuid, request):
         error = OpenRosaResponseBadRequest(
             _(u"Unable to submit because there are multiple forms with"
               u" this formID."))
+    except DataError as e:
+        error = OpenRosaResponseBadRequest(
```

Comment:
Did this have to be 3 lines could, `OpenRosaResponseBadRequest(str(e))` not fit within the 79/80 line limit?

Reply:
Fixed